Dockerfile

```diff
@@ -84,10 +84,12 @@ RUN \
     terraform --version
 
 ENV BOLT_DISABLE_ANALYTICS=true
-ENV PATH=/opt/puppetlabs/server/bin:/opt/puppetlabs/puppet/bin:/opt/puppetlabs/bin:$PATH
+ENV PATH=/opt/puppetlabs/puppet/bin:/opt/puppetlabs/bin:$PATH
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
+COPY Dockerfile /
```

Comment:
what's the purpose of copying it?

Reply:
ah you mentioned it in the PR message.

Reply:
i think its common practice and it is referenced in line 12. will search for a better eplaination.

Reply:
> The `org.label-schema.dockerfile` label specifically indicates the location of the Dockerfile within the image. This can be useful for tools or scripts that inspect Docker images to understand their content and configuration.